src/test/java/net/smartcosmos/dto/metadata/MetadataResponseTest.java

```diff
@@ -70,8 +70,13 @@ public void thatObjectMapperAcceptsNullMetadata() throws JsonProcessingException
         String jsonString = mapper.writeValueAsString(response);
         JSONObject jsonObject = new JSONObject(jsonString);
 
+        // TODO
+        // Due to @JsonInclude(JsonInclude.Include.NON_NULL)
+        // annotation in MetadataResponse, there's no metadata here.
+        // Is this a problem?
```

Comment:
Yeah, IMO `metadata` should be always present. You just need to add `@JsonInclude(JsonInclude.Include.ALWAYS)` to the `metadata` field.